scene/resources/packed_scene.cpp

```diff
@@ -2481,6 +2481,16 @@ Node *PackedScene::instantiate(GenEditState p_edit_state) const {
 	ERR_FAIL_COND_V_MSG(p_edit_state != GEN_EDIT_STATE_DISABLED, nullptr, "Edit state is only for editors, does not work without tools compiled.");
 #endif
 
+	if (state.is_null()) {
+		ERR_PRINT(vformat("Cannot instantiate PackedScene from path '%s': scene state is null. The scene may not have been loaded correctly.", get_path()));
+		return nullptr;
+	}
+
+	if (!state->can_instantiate()) {
+		ERR_PRINT(vformat("Cannot instantiate PackedScene from path '%s': scene state is invalid.", get_path()));
+		return nullptr;
+	}
+
```

Comment:
Left over changes from gltf2 import fix when we had metadata and folder wildcards.
```suggestion
```